examples/parallel_environment.py

```diff
@@ -267,7 +237,7 @@ def parallel_env_sync(
         sim_name=args.sim_name,
         headless=args.headless,
         seed=args.seed,
-        num_agents=args.num_agents,
+        num_agents=args.num_agents + 1,
```

Comment:
Is this intended to always have multi-agent?

Reply:
When a user runs the example, I wanted to make the transition in the envision display between `asynchronous` and `synchronous` approaches obvious. A simple way would be to change the number of agents in each approach. Let me know if you have a better way to clearly show the transition in the envision display.